### ISSUE TITLE
Relationship between hypothesis and prediction

### ISSUE THREAD

**Issue by razvan-panda** [fairy-tale-agi-solutions/fairy-tale-artificial-general-intelligence-solutions#19]
An AGI will probably need to have as part of it a model of reality which it would use to achieve its goals by predicting possible futures in its mind based on its actions. It could make a hypothesis that by performing some set of actions it would arrive at the goal. Because of external factors and/or lack of sufficient predictive capacity the hypothesis could be falsified. In previous phrases hypothesis might be used interchangeably with prediction.

We have to determine the relationship between hypothesis and prediction. Are all hypothesis predictions? Are all predictions hypothesis? Do they intersect?

To figure this out I think you would need to consider three questions:
* are there any hypothesis that are not predictions?
* are there any predictions that are not hypothesis?
* are there any hypothesis that are also predictions?

This ticket might help with work on: https://github.com/fairy-tale-agi-solutions/fairy-tale-artificial-general-intelligence-solutions/issues/16 https://github.com/fairy-tale-agi-solutions/fairy-tale-artificial-general-intelligence-solutions/issues/17

**Comment by MagnusWootton** [comment]
In my english vocabulary, a hypothesis is something that doesnt happen necessarily from now, its an unrelated fantasy,  but a prediction does - and is statisticly based, most probable next event.

**Comment by razvan-panda** [comment]
@MagnusWootton According to google:

hypothesis = a supposition or proposed explanation made on the basis of limited evidence as a starting point for further investigation.

prediction = a thing predicted; a forecast.

supposition = a belief held without proof or certain knowledge; an assumption or hypothesis.

predicted = say or estimate that (a specified thing) will happen in the future or will be a consequence of something.

Based on above it seems that: a hypothesis can be made on the basis of no evidence or limited evidence

Based on what we know about predictions: predictions can be made without any proper evidence backing them up or they are made based on limited evidence.

So it appears that for both terms from point of view of evidence, it can be limited or no evidence.

**Comment by razvan-panda** [comment]
@brandonrobertz Since they are different based on the definitions you used, can you give examples for the following:

* are there any hypothesis that are not predictions?
* are there any predictions that are not hypothesis?
* are there any hypothesis that are also predictions?

Having these examples would make it more likely that what you said is correct.

**Comment by brandonrobertz** [comment]
They're different things is what I've argued. A _prediction_ is a basket term for anything that may or may not happen in the future. A _hypothesis_ is a an explanation of some phenomena, relationship, etc, which can be used to make predictions about the real world.

**Comment by razvan-panda** [comment]
@brandonrobertz Yes, and because you say they are different that implies that at least 1 or both of these sentences have examples:

* are there any hypothesis that are not predictions?
* are there any predictions that are not hypothesis?

**Comment by brandonrobertz** [comment]
Let's look at general relativity as an example:

The hypothesis or theory, from [Wikipedia][1]:

    the curvature of spacetime is directly related to the energy and
    momentum of whatever matter and radiation are present.

This hypothesis has been used to make several predictions (many of which we have detected and confirmed). From [Wikipedia][2]:

    General relativity also predicts novel effects of gravity, such
    as gravitational waves, gravitational lensing and an effect of
    gravity on time known as gravitational time dilation.

> are there any hypothesis that are not predictions?

I would answer yes. The theory of relativity is not a prediction in itself. It describes a relationship between space, time and matter. It's too broad to be a prediction about an event, but it can be used to predict events. This leads us to the second question.

> are there any predictions that are not hypothesis?

Again, I would answer yes. I will give you the 1919 test of general relativity. The prediction was that during a solar eclipse, a photographer directly in the path of totality would be able to photograph a star's light being warped by the Sun's gravity. The result of this prediction is [this image][3].

The prediction about being able to photograph the warped light from a distant star was correct, but was not a hypothesis in its own right.

[1]: https://en.wikipedia.org/wiki/General_relativity
[2]: https://en.wikipedia.org/wiki/History_of_general_relativity
[3]: https://m.eet.com/media/1188473/solar_1919_eclipse_positive.jpg

**Comment by razvan-panda** [comment]
@brandonrobertz I would answer no to both those questions. In my mind I can substitute the words hypothesis and prediction in above examples without any issues. Which makes hypothesis = prediction.

**Comment by razvan-panda** [comment]
For example predictions are based on observation also. Past observations create internal models of reality which help with predicting stuff.

An I do not understand why a hypothesis can seek to explain a phenomenon but a prediction can't. But I am more than just a bit confused by what "to explain" means.

**Comment by brandonrobertz** [comment]
> In my mind I can substitute the words hypothesis and prediction in above examples without any issues. Which makes hypothesis = prediction.

Sure, you could replace the words, but then you're breaking from the meaning of hypothesis as it is defined above by multiple sources from over a hundred years. They are different things in the scientific context, that's all I'm saying.

I've exhausted about all I can say about this subject.

**Comment by razvan-panda** [comment]
I explained in my previous comment why I do not consider being based on observation a difference.
And why can't a prediction explain stuff?

**Comment by MagnusWootton** [comment]
So ur making an ai?
"hypothetical".  It depends how your involving it in your ai machine, is the important thing.

**Comment by razvan-panda** [comment]
@MagnusWootton You could say we are working on figuring out how to create an AGI.
Both the words hypothesis and predictions might be useful for designing an AGI. It could be that the words are useful as they are or that by analyzing their meanings we come to other more basic/orthogonal words that might help with AGI design.

**Comment by razvan-panda** [comment]
@brandonrobertz 

> They are different things in the scientific context, that's all I'm saying.

So if you take both of them and apply them to any context they can be considered equal? If that is true, that makes sense to me - they are different because hypothesis is a prediction used as part of the scientific method usage.

**Comment by razvan-panda** [comment]
@brandonrobertz Regarding the Venn diagram part:

> The theory of relativity is not a prediction in itself. It describes a relationship between space, time and matter. It's too broad to be a prediction about an event, but it can be used to predict events.

This proves nothing relevant because it is talking about a theory not a hypothesis.

> Again, I would answer yes. I will give you the 1919 test of general relativity. The prediction was that during a solar eclipse, a photographer directly in the path of totality would be able to photograph a star's light being warped by the Sun's gravity. The result of this prediction is this image.
> 
> The prediction about being able to photograph the warped light from a distant star was correct, but was not a hypothesis in its own right.

This is like basically saying that a prediction is not a hypothesis because a prediction is not a hypothesis. There is no evidence in the above statements that shows why that prediction is not a hypothesis.

So we are left with no examples of:
* are there any hypothesis that are not predictions?
* are there any predictions that are not hypothesis?

based on that.

**Comment by MagnusWootton** [comment]
To me ->  (You know this is biased crap :))
A hypothesis is something that is 100% true,  but it can be extremely unlikely to happen from now, and its still a valid hypothesis.
A prediction is a hypothesis that you think that is likely to happen next.

So prediction (to me) is inside the total set of hypothesis.

**Comment by razvan-panda** [comment]
@MagnusWootton Because all hypothesis seem to be predictions yet the word hypothesis is only making sense under the scientific method I think that the Venn diagram is of hypothesis being a subset of predictions.

Above is not affected by prediction being more likely to happen since I think we have no reason to think that is the case.

**Comment by brandonrobertz** [comment]
> Im my thinking it could be the other way around, in the fact that hypothesis could be the set of all predictions that dont arise?

This is basically what I'm saying. Hypotheses serve a a framework to make predictions. Therefore a hypothesis is a superset of individual predictions.

**Comment by brandonrobertz** [comment]
This source also explains it in a very basic manner.

[Predicting vs. Hypothesis](https://madaboutscience.weebly.com/prediction-vs-hypothesis.html)

    Prediction:
      A prediction is a guess what might happen based on observation.
    Hypothesis:
      A hypothesis is a possible explanation for a observation or problem
      that can further be tested by experimentation. Hypothesis' are also
      known as educated guesses. 

 Basically, prediction is a specific observable outcome. Hypothesis is about the relationship between observations.

**Comment by razvan-panda** [comment]
Conclusion:
* a hypothesis can be composed of multiple parts that can be supported or refuted individually
* a hypothesis is only valid in the context of the scientific method

**Comment by razvan-panda** [comment]
[Can hypotheses have parts that are supported or falsified individually?
](https://philosophy.stackexchange.com/q/57618/35386)
